app/views/PartTimeHoursView.scala.html

```diff
@@ -26,23 +27,23 @@
         inputNumber: components.inputNumber
 )
 
-@(form: Form[_], partTimePeriod: Period, idx: Int)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], start: LocalDate, end: LocalDate, idx: Int)(implicit request: Request[_], messages: Messages)
 
-@govukLayout(pageTitle = Some(title(form, "partTimeHours.title")), thisPage = PartTimeHoursPage) {
+@govukLayout(pageTitle = Some(title(form, "partTimeHours.title", titleMessageArgs = Seq(dateToString(start), dateToString(end)))), thisPage = PartTimeHoursPage) {
 
     @formHelper(action = PartTimeHoursController.onSubmit(idx)) {
 
         @errorSummary(form.errors)
 
-        @h1(messages("partTimeHours.heading", dateToString(partTimePeriod.end)))
+        @h1(messages("partTimeHours.heading", dateToString(start), dateToString(end)))
```

Comment:
I wonder if we will want to use dateToStringWithoutYear for start so it will read eg. 1 July to 7 July 2020 rather than 1 July 2020 to 7 July 2020